testnets/sgetestnet/chain.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "../chain.schema.json",
   "chain_name": "sgetestnet",
-  "chain_id": "sge-network-3",
+  "chain_id": "sge-network-4",
```

Comment:
With this chain-id change, do the ibc connections remain functional?
https://github.com/cosmos/chain-registry/blob/master/testnets/_IBC/osmosistestnet-sgetestnet.json

Reply:
sge-network-4 is the new testnet, so it should probably be created in testnets/sgetestnet2 and this endpoint should be added there after, correct?